src/components/elements/select-list.tsx

```diff
@@ -148,7 +149,7 @@ const SelectList = ({options = [], label, multiple, ariaLabelledby, required, de
           </div>
         )}
 
-        {optionChosen && <div className="rs-p-3 type-2 max-w-[calc(100%-30px)] overflow-hidden">{renderSelectedValue(value, options)}</div>}
+        {optionChosen ? <div className="rs-p-3 type-2 max-w-[calc(100%-30px)] overflow-hidden">{renderSelectedValue(value, options)}</div> : <span className="type-2">{placeholder}</span>}
```

Comment:
if `placeholder` is `undefined`, this will still output a span. Let's break this into two separate conditions instead of the if/then/else
```
{optionChosen && ...}
{!optionChosen && placeholder && ...}
```